designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -330,4 +345,25 @@ object ProcessesResources {
   case class ProcessNotInitializedError(id: String) extends Exception(s"Scenario $id is not initialized") with NotFoundError
 
   case class NodeNotFoundError(processId: String, nodeId: String) extends Exception(s"Node $nodeId not found inside scenario $processId") with NotFoundError
+
+  case class ProcessesQuery(isSubprocess: Option[Boolean],
+                            isArchived: Option[Boolean],
+                            isDeployed: Option[Boolean],
+                            categories: Option[Seq[String]],
+                            processingTypes: Option[Seq[String]],
+                            names: Option[Seq[String]],
+                           ) {
+    def toRepositoryQuery: FetchProcessesDetailsQuery = FetchProcessesDetailsQuery(
+      isSubprocess = isSubprocess,
+      isArchived = isArchived.orElse(Option(false)), //Back compatibility,
```

Comment:
maybe we can remove this compatibility? It's here for a long time now?

Reply:
Done :+1: